spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -98,7 +103,8 @@ def get_parser():
         metavar=Metavar.int,
         type=int,
         choices=[0, 1],
-        help='Set to 1 to normalize the metrics using a database of healthy controls. Set to 0 to not normalize.'
+        help='Set to 1 to normalize the metrics using a database of healthy controls. Set to 0 to not normalize. '
+             '\nNote: This flag is valid only if the spinal cord segmentation is provided by the flag "-i".',
```

Comment:
I would maybe tweak the wording of the `Note` to explicitly mention when _not_ to use this flag, too:

```suggestion
        help='Set to 1 to normalize the metrics using a database of healthy controls. Set to 0 to not normalize. '
             '\nNote: This flag should not be set to 1 when computing the MCC (i.e. using spinal canal/CSF). '
             'It should only be used when computing the MSCC (i.e. using spinal cord segmentation).',
```

Reply:
good point! agree, done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4331/commits/ed103b5ba154fbc6af8212f91b6355e629ffd748